packages/url/package.json

```diff
@@ -28,7 +28,9 @@
 	"sideEffects": false,
 	"dependencies": {
 		"@babel/runtime": "^7.13.10",
-		"lodash": "^4.17.21",
+		"lodash": "^4.17.21"
+	},
+	"devDependencies": {
```

Comment:
Development dependencies won't install from packages. They need to be listed in the main `package.json` file.